magento/2.4/Dockerfile

```diff
@@ -3,8 +3,8 @@ ARG PHP_VERSION=7.4-fpm
 FROM php:${PHP_VERSION}
 
 LABEL vendor="Artifakt" \
-      author="djalal@artifakt.io" \
-      stage="alpha"
+  author="djalal@artifakt.io" \
```

Comment:
This email adress isn't attributed anymore.
Maybe we should use another one instead ? Like `engineering@artifakt.io` ?

Reply:
Done